quill-core/src/test/scala/io/getquill/quotation/QuotationSpec.scala

```diff
@@ -857,7 +859,7 @@ class QuotationSpec extends Spec {
       }
       "abritrary" in {
         val q = quote(lift(String.valueOf(1)))
-        q.liftings.`java.this.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
+        //q.liftings.`java.this.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
```

Comment:
you can try to replace `String.valueOf(1)` with a tree that doesn't have this package naming issue. Maybe a scala object in the stdlib?